### ISSUE TITLE
Posts getting duplicated in Data view after page reload

### ISSUE THREAD

**Issue by kjmalek** [ushahidi/platform-client-mzima#799]
**Describe the bug**

When viewing posts in the Data view, post are getting duplicated after reloading the page. This behavior does not occur consistently, but it does appear to be related to pagination. Duplication occurs when the pagination parameter is added to the URL (/feed?page=1).

**To Reproduce**

1. Setup a new Ushahidi deployment using the ushahidi/platform-release repository
2. Create a new post
3. View posts in Data view
4. Reload page or navigate to /feed?page=1

**Expected behavior**

Posts should not be duplicated after reloading the page.

**Screenshots**

![ushahidi_duplication](https://github.com/ushahidi/platform-client-mzima/assets/32937586/d0b03115-db88-41e2-a790-ae9208dd8f44)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version: 115.8.0esr

**Comment by codewithmide** [comment]
Can I work on this? I'm from Outreachy

**Comment by maha-sachin** [comment]
Hi @kjmalek I am Mahalakshmi from outreachy intern2024.  I am looking forward to contributing to this project. can I work on this issue?

**Comment by kevincodesa** [comment]
Hello @kjmalek, both backend and frontend setups are complete. I've attempted to replicate the reported bug but have been unsuccessful on my end. Notably, I encountered an issue creating a post, prompting a switch to the production backend API at https://mzima-dev-api.staging.ush.zone/ for testing. This seems to be currently performing as anticipated.


![image](https://github.com/ushahidi/platform-client-mzima/assets/162356104/f1fc0ad6-1c61-4348-9b93-478858389d51)
